ui/components/Sync/SuspendMessageModal.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { Dispatch, SetStateAction } from "react";
+import styled from "styled-components";
+import Button from "../Button";
+import Flex from "../Flex";
+import Modal from "../Modal";
+
+export type Props = {
+  onCloseModal: Dispatch<SetStateAction<boolean>>;
+  open: boolean;
+  setSuspendMessage: Dispatch<SetStateAction<string>>;
+  suspend: any;
+  suspendMessage: string;
+  className?: string;
+};
+
+const MessageModal = styled(Modal)`
+  textarea {
+    width: 100%;
+    box-sizing: border-box;
+    font-family: inherit;
+    font-size: 100%;
+    border-radius: ${(props) => props.theme.spacing.xxs};
+    resize: none;
+    margin-bottom: ${(props) => props.theme.spacing.base};
+  }
+`;
+
+function SuspendMessageModal({
+  className,
+  onCloseModal,
+  open,
+  setSuspendMessage,
+  suspend,
+  suspendMessage,
+}: Props) {
+  const closeHandler = () => {
+    onCloseModal(false);
```

Comment:
`setSuspendMessage("");` here?

Reply:
nice catch I agree that we reset modal if it's closed without suspending